src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicCipherSuitesPolicyTests.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System.Net.Quic.Implementations;
+using System.Net.Security;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Quic.Tests
+{
+    [ConditionalClass(typeof(QuicTestBase<MsQuicProviderFactory>), nameof(IsSupported))]
+    [Collection(nameof(DisableParallelization))]
+    [SkipOnPlatform(TestPlatforms.Windows, "CipherSuitesPolicy is not supported on Windows")]
```

Comment:
This should be working on Windows. We wired it up to Schannel. cc @anrossi

Reply:
This is an issue on .NET layer, all ctors of `CipherSuitesPolicy` throw in Windows

Reply:
That is unfortunate but also the benefit in QUIC is small. This is mostly compat feature IMHO so it works consistently with previous HTTP versions. Can can change the `CipherSuitesPolicy` behavior if there is pressing need.